### ISSUE TITLE
Docker issue not resolved: https://github.com/qnap-dev/container-apps/issues/53

### ISSUE THREAD

**Issue by titan04** [qnap-dev/container-apps#80]
This should have never been closed.  To update a version in docker or to edit the docker yml is still not possible.

**Comment by dskvr** [comment]
Indeed, the developer is lazy as can be inferred from his commit history.

**Comment by s0uls** [comment]
Same issue over here, with latest QTS and container station

**Comment by kozickikarol** [comment]
+1 for possibility to rebuild docker apps

**Comment by redno2** [comment]
still the same, after two years since the first report of that, it is still not possible to upgrade docker via UI, sad

**Comment by dskvr** [comment]
QNAP's inaction on many issues seems to indicate they don't care about customers. But they do a great job at making it known they care about sales. Food for thought.

**Comment by CyberBlaed** [comment]
bump... ;)
